Code/ForceField/MMFF/Params.h

```diff
@@ -20,6 +20,9 @@
 #include <map>
 #include <iostream>
 #include <boost/cstdint.hpp>
+#include <boost/scoped_ptr.hpp>
```

Comment:
To allow us to have some control over warnings, this block of boost includes (including the line above this one) should probably be enclosed in an `#include <RDGeneral/BoostStartInclude.h>` , `#include <RDGeneral/BoostEndInclude.h>` block. An example of this is here: https://github.com/rdkit/rdkit/blob/f4529c910e546af590c56eba01f96e9015c269a6/Code/RDBoost/python.h#L1